templates/search_results.hbs

```diff
@@ -33,18 +36,25 @@
       {{/if}}
       {{#if subfilters}}
         <section class="filters-in-section collapsible-sidebar">
+          <button type="button" class="collapsible-sidebar-toggle" aria-expanded="false"></button>
```

Comment:
Same here.

Reply:
this one is also needed to expand subfilters on mobile web